quinn-udp/tests/tests.rs

```diff
@@ -10,8 +10,12 @@ use socket2::Socket;
 
 #[test]
 fn basic() {
-    let send = UdpSocket::bind("[::1]:0").unwrap();
-    let recv = UdpSocket::bind("[::1]:0").unwrap();
+    let send = UdpSocket::bind("[::1]:0")
+        .or_else(|_| UdpSocket::bind("127.0.0.1:0"))
+        .unwrap();
+    let recv = UdpSocket::bind("[::1]:0")
+        .or_else(|_| UdpSocket::bind("127.0.0.1:0"))
+        .unwrap();
```

Comment:
Could we just use IPv4 here (and below) unconditionally for simplicity?

Reply:
I think keeping it like that make tests more robust, but i am fine to use ipv4 only if you want.

Reply:
Sure, I'm happy either way. It's pretty terse after all.